assets/xml/objects/object_ddan_objects.xml

```diff
@@ -0,0 +1,12 @@
+<Root>
+    <File Name="object_ddan_objects" Segment="6">
+        <DList Name="gDdanObjectsDL0" Offset="0xC0"/>
+        <DList Name="gDdanObjectsDL1" Offset="0x1F0"/>
+        <DList Name="gDdanObjectsDL2" Offset="0x1350"/>
+        <DList Name="gDdanObjectsDL3" Offset="0x37B8"/>
+        <DList Name="gDdanObjectsDL4" Offset="0x48A8"/>
+        <Collision Name="gDdanObjectsCol0" Offset="0x1DDC"/>
+        <Collision Name="gDdanObjectsCol1" Offset="0x3CE0"/>
+        <Collision Name="gDdanObjectsCol2" Offset="0x4F30"/>
+    </File>
+</Root>
```

Comment:
I can't get palettes to not be like that i tried for a long time and nothing worked..see what i have now and tell me what u think

Reply:
For now, ZAPD apparently can't extract larger palettes (this will be fixed soon). For now extract as 16 * 16 and leave a comment. ("Palette is actually 258 colours" or something.)